ipaserver/install/replication.py

```diff
@@ -600,7 +601,20 @@ def finalize_replica_config(self, r_hostname, r_binddn=None,
             r_conn.simple_bind(r_binddn, r_bindpw)
         else:
             r_conn.gssapi_bind()
-        self._finalize_replica_settings(r_conn)
+        # If the remote server has 389-ds < 1.3, it does not
+        # support the attributes we are trying to set.
+        # Find which 389-ds is installed
+        rootdse = r_conn.get_entry(DN(''), ['vendorVersion'])
+        version = rootdse.single_value.get('vendorVersion')
```

Comment:
Does ``get_entry`` return bytes or text? The regular expression will fail under Python 3 if the value is bytes.

Reply:
get_entry returns an object `ipapython.ipaldap.LDAPEntry` and single_value.get() returns a str in python3, a unicode in python2, so the reg exp should not have any issue.